### ISSUE TITLE
[Bug] Investigate `txm` version 8+ compatibility

### ISSUE THREAD

**Issue by cycraig** [iotaledger/identity.rs#515]
## Bug description
The Wasm bindings README.md examples do not work with the latest version of `txm` (version 8.0.0 currently).

Last working version is 7.4.7

Likely related to the ESM migration stuff done in https://github.com/anko/txm/releases/tag/v8.0.0

## IOTA Identity version
Which version of IOTA Identity are you using?

- Version: dev branch, currently commit 9616bab399211093562e2cfb51b223a70efe8333

## Hardware specification

- Operating system: Ubuntu 20.04 / WSL

## Steps To reproduce the bug
Explain how the maintainer can reproduce the bug.

1. Install `txm` version 8+: `npm install -g -f txm@8` or `npm install -g -f txm@latest`
2. Navigate to the Wasm bindings directory: `cd bindings/wasm`
3. Run: `txm README.md`

## Expected behaviour

Runs successfully with the following output:

```
TAP version 13
1..1
ok 1 Nodejs Example

# 1/1 passed
# OK
```

## Actual behaviour

Throws an error.

## Errors
Error when running `txm README.md` from the `bindings/wasm` directory with version 8.0.0 (`txm version` gives the same error):
```
file:///C:/Users/cycra/AppData/Roaming/npm/node_modules/txm/src/cli.js:16
const programName = Object.entries(packageMetadata.bin)
                           ^

TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at file:///C:/Users/cycra/AppData/Roaming/npm/node_modules/txm/src/cli.js:16:28
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12) 
```

**Comment by cycraig** [comment]
Related to https://github.com/iotaledger/identity.rs/pull/513 where `txm` was pinned to version 7+.

**Comment by cycraig** [comment]
Seems to be fixed by `txm` 8.0.1.
